js-exercises/alphabetic-shift/alphabeticShift.js

```diff
@@ -0,0 +1,9 @@
+const alphabeticShift = string => {
+  const shiftedString = [];
+  for(let char of string){
```

Comment:
`let char`
This doesn't need to be `let`. Because it's never assigned again.

Reply:
Yeah got it  thanks